babel/src/board.js

```diff
@@ -283,20 +280,29 @@ export class Board {
         let hasWideStyleMatch = _.some(_.map(flatChunks, hasMatchInSingleRow));
         return hasWideStyleMatch || _.some(_.map(chunks), hasMatchInPairOfRows);
     }
+    
+    hasMatchEvent() {
+        if (findMatches(this.orbs)[0]) { return true };
```

Comment:
You could force it to boolean, and not miss the `false` part (which you're missing).

```js
return Boolean(findMatches(this.orbs)[0]);
```

which is what I prefer. You're asking a question, "does this have a match event"? The answer should be yes/no (true/false).